libs/api/prisma/migrations/20221229154844_unwrap_array_of_jsonb/migration.sql

```diff
@@ -0,0 +1,84 @@
+-- Migrate article revision blocks from jsob[] to jsonb
```

Comment:
Reason for this was that in the database it was an array of json objects instead of a JSON array.
It's unnecessary for it to be this way and it makes it harder to work with in SQL.